src/cai_lab_to_nwb/zaki_2024/interfaces/zaki_2024_edf_interface.py

```diff
@@ -14,6 +16,10 @@ def __init__(self, file_path: FilePath, verbose: bool = False):
         self.verbose = verbose
         super().__init__(file_path=file_path)
 
+    def get_timestamps_reference_time(self):
```

Comment:
`get_session_start_time` to be in line with the other methods. I don't remember this is "means" object already a datetime object?

Reply:
I wanted to differentiate from the session start time because this would not be the session start time for this conversion. It's just an auxiliary function to get the time reference of the first timestamp of these traces. I will use this to slice the traces once we understand with the author how they want to align this data.

Reply:
This is the case for all the other interfaces. We are never sure if they will be the session_start_time, it is only when you use that interface as stand-alone that it should work that way.

OK, but that makes sense.  We will probably not generalize this code outside of this conversion.